pkg/env/continuous_profiling.go

```diff
@@ -0,0 +1,18 @@
+package env
+
+var (
+	// ContinuousProfiling indicates if continuous profiling is enabled
+	ContinuousProfiling = RegisterBooleanSetting("ROX_CONTINUOUS_PROFILING", false)
+
+	// ContinuousProfilingServerAddress defines the server address for the continuous profiling
+	ContinuousProfilingServerAddress = RegisterSetting("ROX_CONTINUOUS_PROFILING_SERVER_ADDRESS")
```

Comment:
Are there any reasonable defaults possible? Maybe something like `pyroscope.svc.local:<default port>`

Reply:
I've set the default to `http://pyroscope.stackrox.svc.cluster.local.:4040` which is the default service created if you install pyroscope in the stackrox namespace.